filter/big_filter/filtertobig/filter_to_big.go

```diff
@@ -0,0 +1,8 @@
+package filtertobig
+
+import "math/big"
+
+// FilterToBig defines interface between int to big.Int pointer
+type FilterToBig interface {
```

Comment:
I don't really like the combination of conversion and filtering.

Idea: create `ConvertTo` and `ConvertFrom` functions that perform the conversion, and use consistent size throughout.